Makefile

```diff
@@ -146,11 +145,8 @@ test-unit-parallel: $(PARATEST) vendor
 
 .PHONY: test-unit-docker
 test-unit-docker:	## Runs the unit tests on the different Docker platforms
-test-unit-docker: test-unit-80-docker
-
-.PHONY: test-unit-80-docker
-test-unit-80-docker: $(DOCKER_FILE_IMAGE) $(PHPUNIT)
-	$(DOCKER_RUN_80) $(PHPUNIT) --group $(PHPUNIT_GROUP)
+test-unit-docker:
+	@echo "No test currently registered"
```

Comment:
didn't get this, what does it mean? Can't we run just tests with docker for PHP 8.1?

Reply:
looks like you missed this comment :)

Reply:
I was thinking we were doing this for older versions only since I saw there was already PHP 8.1 related constants, but no docker tests for 8.1.

If that's the not the case then it makes sense to restore it and add PHP 8.2 as well

Reply:
reverted https://github.com/infection/infection/pull/1839